ros2bag/CHANGELOG.rst

```diff
@@ -2,8 +2,8 @@
 Changelog for package ros2bag
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Forthcoming
```

Comment:
It seems like CHANGELOG is outside the scope of this feature, shouldn't we be generating when we create the next release? Same for the other CHANGELOGs